packages/patternlab/styleguide/source/_patterns/02-molecules/press-teaser.twig

```diff
@@ -2,14 +2,18 @@
 
 <section class="ma__press-teaser {{ imageClass }}">
   {% if pressTeaser.image %}
-    <a 
+    <a
       class="ma__press-teaser__image"
-      href="{{ pressTeaser.title.href }}" 
+      href="{{ pressTeaser.title.href }}"
+    {% if pressTeaser.title.text %}

```

Comment:
I think we don't need the check here because the title link text should be a required prop, instead of optional. Without the link text, this component is not useful.